### PR TITLE
[Notifier] Add Smsbox notifier bridge

### PR DESCRIPTION
**Pull request by alanzarli**

### CODE REVIEW

**Review by OskarStark — CHANGES_REQUESTED** [review]
Please add a `.. versionadded::` directive, you can check the same page on 6.4 branch for an example

**Review by OskarStark — APPROVED** [review]

File: notifier.rst [hunk 1]
```diff
@@ -92,6 +92,7 @@ Service             Package                                DSN
 `Sinch`_            ``symfony/sinch-notifier``             ``sinch://ACCOUNT_ID:AUTH_TOKEN@default?from=FROM``
 `Smsapi`_           ``symfony/smsapi-notifier``            ``smsapi://TOKEN@default?from=FROM``
 `SmsBiuras`_        ``symfony/sms-biuras-notifier``        ``smsbiuras://UID:API_KEY@default?from=FROM&test_mode=0``
+`Smsbox`_           ``symfony/smsbox-notifier``            ``smsbox://APIKEY@default?mode=MODE&strategy=STRATEGY&sender=SENDER``
```
Comment (alamirault) [inline notifier.rst:95]: Link declaration must be added at end of file

After
https://github.com/symfony/symfony-docs/blob/9ccbd0c7b572c2bf1b64fe4e9109566376ecba3a/notifier.rst?plain=1#L1003

**Review by alamirault — CHANGES_REQUESTED** [review]

**Comment by OskarStark** [comment]
Thank you Alan.

File: notifier.rst (hunk 1 above)
Comment (OskarStark) [reply notifier.rst:95]: Done in fc23fa362

**Pull request merged**